sites/partners/src/pages/_app.tsx

```diff
@@ -20,6 +25,25 @@ function BloomApp({ Component, router, pageProps }: AppProps) {
   const { locale } = router
   const skipLoginRoutes = ["/forgot-password", "/reset-password", "/users/confirm", "/users/terms"]
 
+  const fileProviderConfig: FileProviderConfig = {
+    publicService: {
+      fileServiceType: FileServiceTypeEnum.cloudinary,
```

Comment:
shouldn't this also depend on a env variable instead of being hardcoded?

Reply:
I'm going to add that in a future PR once we have more than one option for file service, and read what file service to use from an env variable.